### DISCUSSION TITLE
How to setup this development environment locally? please help.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/fluentui#25141] [General]
I want to contribute. But I don't know how to setup this locally. I didn't find any contribute guide. So please, anyone help me how to setup this locally?

**Comment by khmakoto** [comment]
Hi @iSatVeerSingh, please take a look at our [setup wiki](https://github.com/microsoft/fluentui/wiki/Setup#basic-setup) and let us know if there is anything in there that is confusing or any additional information that you would expect to find there. Thanks!

**Reply by samavati** [reply]
Hi @khmakoto I have read the setup wiki page and followed all the steps, but when I run `yarn start `and then choose the `@fluentui/docs` I get this error:

```bash
ERROR in ../react-northstar/src/components/Portal/PortalInner.tsx 12:0-72
Module not found: Error: Can't resolve '@fluentui/react-portal-compat-context' in '/Users/ehsansamavati/Documents/Work/fluentui/packages/fluentui/react-northstar/src/components/Portal'
```
and also I can't access anything on localhost:8080 could you please help me with this?

**Comment by micahgodbolt** [comment]
@iSatVeerSingh what package are you wanting to contribute to? @fluentui/docs is the Northstar Teams project. If you want to contribute to v8 choose @fluentui/react or v9 @fluentui/react-components.
